webview/src/experiments/components/table/Row.tsx

```diff
@@ -92,6 +96,46 @@ const getMultiSelectMenuOptions = (selectedRowsList: RowProp[]) => {
       MessageFromWebviewType.REMOVE_EXPERIMENT,
       hideRemoveOption,
       true
+    ),
+    {
+      divider: true,
+      id: 'clear-selection',
+      keyboardShortcut: 'Esc',
+      label: 'Clear row selection'
+    }
+  ]
+}
+
+const getWorkspaceOptions = (
+  withId: (
+    label: string,
+    type: MessageFromWebviewType,
+    hidden?: boolean,
+    divider?: boolean
+  ) => MessagesMenuOptionProps,
+  isWorkspace: boolean,
+  projectHasCheckpoints: boolean,
+  hideVaryAndRun: boolean,
+  depth: number
+) => {
+  const isNotCheckpoint = depth <= 1 || isWorkspace
+
+  return [
+    withId(
+      projectHasCheckpoints ? 'Modify and Resume' : 'Modify and Run',
+      MessageFromWebviewType.VARY_EXPERIMENT_PARAMS_AND_RUN,
+      !isNotCheckpoint,
+      !hideVaryAndRun
+    ),
+    withId(
+      'Modify, Reset and Run',
```

Comment:
[Q] Can we reorder these to match the tree context menu? 

<img width="242" alt="image" src="https://user-images.githubusercontent.com/37993418/177448072-622641c7-30fc-40e1-9286-66c22c4fc46a.png">

Reply:
The reason being "Reset and Run" is the most likely candidate for the next action.

Reply:
Yes, that seems like a simple change